types/comtac/Cluey/uplink.js

```diff
@@ -147,11 +220,11 @@ function decodeUplink(input) {
       if (input.fPort === PING_PORT) {
         return decodePingPayload(input.bytes);
       }
-      errors.push("not a valid port");
+      errors.push("INVALID_PORT");
```

Comment:
To be able to modify this string more easily in the future, you could introduce a `const` that you than can reuse as a variable everywhere in your code where this value is used. This helps you modify its value at a single place, but then gets reflected everywhere. This applies to other values as well, such as f.e. `UNKNOWN`). E.g.

```
const INVALID_PORT = "INVALID_PORT";
...
errors.push(INVALID_PORT);
```